### PR TITLE
Feature/button component

### PR DESCRIPTION
**Pull request by christian-cg**
[Trello ticket](https://trello.com/c/CMgwuvKN/25-button-component-2-as-a-user-i-need-buttons-that-are-visually-appealing-and-confortable-tu-use)

* Added styles for all the buttons to be used across the site

Screen Shot 2019-03-28 at 6 12 39 PM

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,5 +1,63 @@
 /* General page styles */
 
-
 /* We should use a mobile first approach having out breack point at 768px */
 
+.button {
+	font-family: roboto, sans-serif;
+	font-weight: 500;
+	border-radius: 2px;
+	overflow: hidden;
```
Comment (camigomezdev) [inline css/style.css:9]: Why did you set this property?

File: index.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
```
Comment (jusk02) [inline index.html:1]: Please undo all changes to this file as they don't serve any purpose and generate noise.

File: css/style.css (hunk 1 above)
Comment (christian-cg) [reply css/style.css:9]: That was to avoid bad behavior of the button when the user attempt content that overflows the designed width of it.

**Review by camigomezdev — APPROVED** [review]
💪😄

File: index.html (hunk 2 above)
Comment (christian-cg) [reply index.html:1]: I will keep the third (empty) line for better visualization. The other line linking the stylesheet was removed.

**Review by jusk02 — APPROVED** [review]

**Pull request merged**